content_creator.py

```diff
@@ -146,9 +149,26 @@ def copy_test_files(bundle_test):
             shutil.copyfile(path, os.path.join(bundle_test, os.path.basename(path)))
 
 
-def main(circle_artifacts):
-    print('Starting to create content artifact...')
+def update_content_version(content_ver: str):
+
+    path = './Scripts/CommonServerPython/CommonServerPython.py'
+    regex = r'CONTENT_RELEASE_VERSION = .*'
+    try:
+        with open(path, 'r+') as f:
+            content = f.read()
+            content = re.sub(regex, f"CONTENT_RELEASE_VERSION = '{content_ver}'", content, re.M)
+            f.seek(0)
+            f.write(content)
+    except Exception as ex:
+        print_warning(f'Could not open CommonServerPython File - {ex}')
 
+
+def main(circle_artifacts, content_version):
```

Comment:
```suggestion
def main(circle_artifacts, content_version, *args, **kwargs):
```
we don't know if someone will pass something into the file, so that's a good way to prevent errors.